java-checks/src/main/java/org/sonar/java/checks/helpers/SyntaxNodePredicates.java

```diff
@@ -37,4 +38,13 @@ public boolean apply(Tree syntaxNode) {
     };
   }
 
+  public static Predicate<String> isSubtypeOf(final Type type) {
+    return new Predicate<String>() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this a named "static" inner class. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS2694)